### PR TITLE
[7.3.x] GUVNOR-3427: Add descriptions to new GDT asset form

### PR DESCRIPTION
**Pull request by jomarko**
screenshot from 2017-08-24 22-42-59

Cherry-pick of #589

### CODE REVIEW

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/handlers/GuidedDecisionTableOptions.ui.xml [hunk 1]
```diff
@@ -23,13 +23,21 @@
...
-    <b:CheckBox text="{i18n.UseWizardToBuildAsset}" ui:field="chkUseWizard"></b:CheckBox>
-    <handlers:HitPolicySelector ui:field="hitPolicySelector"></handlers:HitPolicySelector>
-    <gwt:VerticalPanel>
-      <b:Radio name="tableFormat" text="{i18n.TableFormatExtendedEntry}" ui:field="optExtendedEntry" value="true"/>
-      <b:Radio name="tableFormat" text="{i18n.TableFormatLimitedEntry}" ui:field="optLimitedEntry"/>
-    </gwt:VerticalPanel>
-  </gwt:VerticalPanel>
+  <b:Form>
+    <b:FormGroup>
+      <b:CheckBox text="{i18n.UseWizardToBuildAsset}" ui:field="chkUseWizard" title="Create table with help of the wizard" />
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/handlers/GuidedDecisionTableOptions.ui.xml]: See https://github.com/kiegroup/drools-wb/pull/589#pullrequestreview-59167689

**Comment by jomarko** [comment]
jenkins please retest this

@manstis thanks, updated

**Comment by manstis** [comment]
Thanks @jomarko 

Please let me know when green and I'll merge.

**Comment by jomarko** [comment]
@manstis green

**Comment by jomarko** [comment]
@manstis @Rikkola same as #589, please have a look when you have a moment.

**Pull request merged**